lib/matplotlib/text.py

```diff
@@ -436,6 +436,8 @@ def _get_layout(self, renderer):
                 offsety = (ymin + height)
             elif valign == 'baseline':
                 offsety = (ymin + height) - baseline
+            elif valign == 'center_baseline':
+                offsety = (ymin + (height + (height - baseline)) / 2.0)
```

Comment:
In other words, `ymin + height - baseline / 2`.  This is the same as 'top' minus `baseline / 2` which makes sense, since baseline is the height starting from the baseline rather than the descender.  Right?